walkthroughs/eks-app-mesh-cross-account-acm/infrastructure/infrastructure_backend.yaml

```diff
@@ -0,0 +1,376 @@
+      - IpProtocol: "-1"
+        CidrIp: "10.192.0.0/16"
+      VpcId: !Ref VPC
+
+  VPCPeeringConnection:
+    Type: 'AWS::EC2::VPCPeeringConnection'
+    Properties:
+      VpcId: !Ref VPC
+      PeerVpcId: !Ref PeerVPCId
+      PeerOwnerId: !Ref FrontendAccountId
+      PeerRoleArn: !Ref PeerRoleArn
+
+  NodesSDPolicy:
+    Type: AWS::IAM::ManagedPolicy
+    Properties:
+      PolicyDocument:
+        Version: 2012-10-17
+        Statement:
+        - Effect: Allow
+          Action: ['appmesh-preview:*']
+          Resource: '*'
+        - Effect: Allow
+          Action: ['servicediscovery:CreateService']
+          Resource: '*'
+        - Effect: Allow
+          Action: ['servicediscovery:DeleteService']
+          Resource: '*'
+        - Effect: Allow
+          Action: ['servicediscovery:GetService']
+          Resource: '*' 
+        - Effect: Allow
+          Action: ['servicediscovery:GetInstance']
+          Resource: '*' 
+        - Effect: Allow
+          Action: ['servicediscovery:RegisterInstance']
+          Resource: '*'
+        - Effect: Allow
+          Action: ['servicediscovery:DeregisterInstance']
+          Resource: '*'
+        - Effect: Allow
+          Action: ['servicediscovery:ListInstances']
+          Resource: '*'
+        - Effect: Allow
+          Action: ['servicediscovery:ListNamespaces']
+          Resource: '*' 
+        - Effect: Allow
+          Action: ['servicediscovery:ListServices']
+          Resource: '*' 
+        - Effect: Allow
+          Action: ['servicediscovery:GetOperation']
+          Resource: '*'
+        - Effect: Allow
+          Action: ['servicediscovery:GetInstancesHealthStatus']
+          Resource: '*'
+        - Effect: Allow
+          Action: ['servicediscovery:UpdateInstanceCustomHealthStatus']
+          Resource: '*'
+        - Effect: Allow
+          Action: ['route53:GetHealthCheck']
+          Resource: '*' 
+        - Effect: Allow
+          Action: ['route53:CreateHealthCheck']
+          Resource: '*'
+        - Effect: Allow
+          Action: ['route53:UpdateHealthCheck']
+          Resource: '*' 
+        - Effect: Allow
+          Action: ['route53:ChangeResourceRecordSets']
+          Resource: '*' 
+        - Effect: Allow
+          Action: ['route53:DeleteHealthCheck']
+          Resource: '*' 
+  
+  AppServerServiceDiscoveryNamespace:
+    Type: AWS::ServiceDiscovery::HttpNamespace
+    Properties: 
+      Name: !Sub '${ProjectName}.local'
+
+  # AppServerServiceDiscoveryNamespace:
+  #   Type: AWS::ServiceDiscovery::PrivateDnsNamespace
+  #   Properties:
+  #     Name: !Sub '${ProjectName}.local'
+  #     Vpc: !Ref VPC
+
+  # BackendRegistry:
+  #   Type: AWS::ServiceDiscovery::Service
+  #   Properties:
+  #     Name: 'yelb-appserver'
+  #     DnsConfig:
+  #       NamespaceId: !Ref AppServerServiceDiscoveryNamespace
+  #       DnsRecords:
+  #       - Type: A
+  #         TTL: 300
+  #     HealthCheckCustomConfig:
+  #       FailureThreshold: 1
```

Comment:
If there are unused bits of configuration, please remove them.

Reply:
This is needed for the appserver to connect with the back end modules - redis and Postgres

Reply:
Does the walkthrough involve un-commenting this yaml and updating the stack? It's _much_ friendlier to use a parameter and a condition.

Reply:
yelb app setup doesnt require the end user to un-comment anything in the files.

Reply:
So then, what are these commented-out resources doing?

Reply:
Dont need the commented out resources, cleaning it up.

Reply:
Done with the clean-up.